scripts/plugins/storage/postgres/init.sql

```diff
@@ -760,6 +760,17 @@ CREATE TABLE foglamp.plugin_data (
 	data    jsonb                     NOT NULL DEFAULT '{}'::jsonb,
 	CONSTRAINT plugin_data_pkey PRIMARY KEY (key) );
 
+-- Create filters table
+CREATE TABLE foglamp.filters (
+             name        character varying(255)        NOT NULL,
+             plugin      character varying(255)        NOT NULL,
+       CONSTRAINT filter_pkey PRIMARY KEY( name ) );
+
+-- Create filter_users table
+CREATE TABLE foglamp.filter_users (
+             name        character varying(255)        NOT NULL,
+             user        character varying(255)        NOT NULL);
```

Comment:
`user` is a reserved key here. **"user"** this will work.
https://stackoverflow.com/questions/10891368/postgres-table-column-name-restrictions